internal/core/window.rs

```diff
@@ -351,8 +350,13 @@ impl WindowInner {
                 default_font_size_prop.set(window_adapter.renderer().default_font_size());
             }
         }
-        window_adapter.request_window_properties_update();
         window_adapter.request_redraw();
+        let weak = Rc::downgrade(&window_adapter);
```

Comment:
I changed this to capture a weak.

My theory is that the global destructor destructs the closure which destruct the window_adapter which was kept alive, and possibly this causes the crash.

Reply:
Ahhh - well spotted!